core/domain/user_jobs_one_off.py

```diff
@@ -362,9 +362,32 @@ def map(user_model):
         user_model.put()
 
 
-class CleanupActivityIdsFromUserSubscriptionsModelOneOffJob(
+class FillExplorationIdsInUserSubscriptionsModelOneOffJob(
         jobs.BaseMapReduceOneOffJobManager):
-    """One off job that removes nonexisting activity ids from
+    """One off job that copies from activity_ids to exploration_ids
+    in UserSubscriptionsModel.
+    """
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        """Return a list of datastore class references to map over."""
+        return [user_models.UserSubscriptionsModel]
+
+    @staticmethod
+    def map(model_instance):
+        model_instance.exploration_ids = model_instance.activity_ids
+        model_instance.update_timestamps(update_last_updated_time=False)
```

Comment:
If it's important to ensure the last updated time isn't changed, I suggest explicitly verifying that in the migration job tests.

Reply:
Done.